astronify/series/series.py

```diff
@@ -144,7 +147,18 @@ def data(self):
 
     @data.setter
     def data(self, data_table):
-        assert isinstance(data_table, Table), 'Data must be a Table.'
+
+        if not isinstance(data_table, Table):
+            raise TypeError('Data must be an astropy.table.Table object.')
+
+        for c in list(data_table.columns):
+            data_table.rename_column(c, c.lower())
+
+        if "time" not in data_table.columns:
+            raise AttributeError("Input Table must contain a column 'time'")
+
+        if "flux" not in data_table.columns:
```

Comment:
You can set different time/flux collumns using `time_col` and `val_col` arguments to the class constructor, so those shoudl be the column names checked against here.

Reply:
Done! :)